nixos/modules/programs/streamcontroller.nix

```diff
@@ -14,7 +14,10 @@ in
   };
 
   config = lib.mkIf cfg.enable {
-    environment.systemPackages = [ cfg.package ];
+    environment.systemPackages =
+      [ cfg.package ]
```

Comment:
Should we add python interpreter here as well? If yes, how do we make sure we have the same python? Via `passthru`?

Reply:
I think `python` and `kdotool` should not be in `environment.systemPackages` as global `PATH` would be polluted. Would it work if we just add them to `PATH` in the `makeWrapper` in `pkgs/by-name/st/streamcontroller/package.nix`?

Then add `isKde` as an option of `pkgs/by-name/st/streamcontroller/package.nix`, and add `pkgs.streamcontroller.override { isKde =  config.services.desktopManager.plasma6.enable; }` to `environment.systemPackages` here

Reply:
You are correct, I've changed the PR to reflect this but I'm not sure if this is much nicer?